doc/Release_Build_Creation.rst

```diff
@@ -1,16 +1,35 @@
-=======================
-Release Build Creation
-=======================
+=========================================================
+Compile & Build Complete Cortx Stack Via Docker Container
+=========================================================
 
 This file consists of the procedure that should be followed to generate the release build outside the Seagate network using `cortx-build <https://github.com/orgs/Seagate/packages/container/package/cortx-build>`_ docker image. 
 
-***************
-Procedure
-***************
+*****************************************
+Specifications to setup Virtual Machine
+*****************************************
 
-#. Setup a `CentOS 7.8.2003 <http://isoredirect.centos.org/centos/7.8.2003/isos/x86_64/>`_ system.
+- Single-Node VM deployment
```

Comment:
If this information is for single-node VM deployment then remove the per node information. It will confuse the users. Per node information usually written for multi-node deployment.

Reply:
yes, done